contracts/schemes/CL4RRedeemer.sol

```diff
@@ -0,0 +1,146 @@
+pragma solidity ^0.6.12;
+// SPDX-License-Identifier: GPL-3.0
+
+import "./ContinuousLocking4Reputation.sol";
+
+/**
+ * @title A scheme for redeeming a ContinuousLocking4Reputation contract used in a different DAO.
+ * This is used to migrate from an older DAO to a new one while the older DAO has an active ContinuousLocking4Reputation
+ * Using it should be done by initiaalizing this contract s a scheme in the new DAO,
+ * with the old ContinuousLocking4Reputation as an init parameter.
+ * Then the old ContinuousLocking4Reputation should be unregistered as a scheme from the older DAO so
```

Comment:
Note : the old ContinuousLocking4Reputation should be unregistered as a scheme from the older  ....